apps/bmd/prodserver/index.js

```diff
@@ -17,13 +17,9 @@ app.use((req, res, next) => {
 })
 proxy(app)
 
-app.get('/machine-config', (req, res) => {
-  res.json({
-    "appMode": process.env.VX_APP_MODE || "VxMark",
-    "machineId": process.env.VX_MACHINE_ID || "000",
-  })
-})
-
```

Comment:
This was already applied by the `proxy(app)` call, so there's no need to duplicate it here.